src/components/ui/IconButton.js

```diff
@@ -33,12 +38,13 @@ const StyledIconButton = styled.button`
   }
 `
 
-const IconButton = ({ size, raised, icon, onClick, label }) => (
+const IconButton = ({ size, raised, icon, onClick, label, pressed }) => (
   <StyledIconButton
     aria-label={label}
     size={size}
     raised={raised}
     onClick={onClick}
+    pressed={pressed}
   >
```

Comment:
StyledIconButton doesn't need to be a wrapper, you can just return the styled component directly.

Reply:
Doesn't it need to be a wrapper because of the icon child?

Reply:
We can change it to children which I think is cleaner. I did something similar with Button, it had an icon prop but I just consolidated with children.